SPEC.md

```diff
@@ -700,8 +702,8 @@ JSON Schema for the Image Manifest (app image manifest, ACI manifest), conformin
     },
     "dependencies": [
         {
-            "app": "example.com/reduce-worker-base",
-            "imageID": "sha512-...",
+            "name": "example.com/reduce-worker-base",
+            "id": "sha512-...",
```

Comment:
Let's revert this to "imageID". Right now it does mean something rather strict. If we want to loosen the requirement we can address that separately.